src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -381,6 +381,29 @@ public void testQueryStatuses() throws SQLException, IOException, InterruptedExc
     assertEquals(QueryStatus.QUEUED, status);
   }
 
+  /**
+   * When 1) there is an underscore in account_name; 2) the account name can only be inferred from
+   * connection string; 3) externalbrowser is used, backend failed to parse the correct account
+   * name. See SNOW-496117 for more details. Don't run on github because the github testing
+   * deployment is likely not having the test account we used here.
+   */
+  @Test
+  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
+  public void testExternalBrowserWithUnderscoreInAccountName() throws SQLException {
```

Comment:
With the fix in SNOW-496117, shouldn't this be succeeding? This appears to be testing a failure case that's the result of us failing to parse the name as expected. But with the fix in place, this should work, no?

Reply:
Please ignore this test. I just came up with a new test.